app/views/listings/new.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1 Create new listing
+
+
+= form_with model: @listing, local: true do |form|
+  %p
+    = form.label :Name
+    %br/
+    = form.text_field :name
+  %p
+    = form.label :Location
+    %br/
+    = form.text_field :location
+  %p
+    = form.label :Description
+    %br/
+    = form.text_area :description  
+  %p
+    = form.label :Availability
+    %br/
+    = form.text_field :availability
+  %p
+    = form.label :Email
```

Comment:
please use downcase here

Reply:
Is the naming standard same for labels, as for field names and id's?

Reply:
The form builder will use the attribute name and `humanize` it. It means that `:email` will be rendered as `Email` for this label.

Reply:
the upper case is for the labels. we want it to display that way.

Reply:
See my comment above and try it out in a browser.

Reply:
done. thanks.